PWA/src/components/Timesheet/ConfirmSubmission.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-row justify="center">
+    <v-btn color="success" dark @click.stop="dialog = true" @click="validate">
+      Submit
+    </v-btn>
+
+    <v-dialog v-model="dialog" max-width="300">
+      <div v-if="valid">
+        <v-card>
+          <div v-if="!loading">
+            <v-card-title class="headline" id="confirm"
+              >Are you sure want to submit the form?</v-card-title
+            >
+
+            <v-card-text>
+              Some text talking about how this submission is final unless
+              something is wrong with it.
+            </v-card-text>
+
+            <v-card-actions>
+              <v-spacer></v-spacer>
+
+							<!-- Confirm if user is ready to submit -->
+              <v-btn color="red" text @click="dialog = false">
+                Cancel
+              </v-btn>
+              <v-btn color="green darken-1" text @click="submit">
+                Submit
+              </v-btn>
+            </v-card-actions>
+          </div>
+
+          <div v-else>
+            <div v-if="!returnHome">
+
+							<!-- Submitting the form -->
+              <div class="text-center">
+                <v-progress-circular
+                  :size="50"
+                  color="primary"
+                  indeterminate
+									id="progress"
+                ></v-progress-circular>
+                <p class="text--disabled">Submitting form</p>
+              </div>
+            </div>
+
+            <div v-else>
+						<!-- Display submission status -->
+              <div v-if="submissionStatus">
+                <v-card-title class="headline text-center" id="submited"
+                  >Your form has been submitted!</v-card-title
+                >
+
+                <v-card-text class="text-center" id="submissionError">
+                  Some text on what will come next for the employee.
+                </v-card-text>
+              </div>
+              <div v-else>
+                <v-card-title class="headline" id="failure"
+                  >Something has gone wrong</v-card-title
+                >
+
+                <v-card-text>
+                  Please try again.
+                </v-card-text>
+              </div>
+            </div>
+          </div>
+        </v-card>
+      </div>
+				
+			<!-- The form is not valid -->
```

Comment:
Indentation is off

Reply:
As discussed with Keaton, this is an issue regarding github displaying tabs as 8 spaces.